string name
string[] variables
uint32 timeStep
